packages/admin-next/admin-shared/src/extensions/virtual/utils.ts

```diff
@@ -0,0 +1,29 @@
+import { InjectionZone } from "../widgets"
+
+const PREFIX = "virtual:medusa/"
+
+export const id = (name: string) => {
```

Comment:
nitsuggestion: id sounds like a value but it is a function, could we name it like resolveId for example?